sda-commons-server-weld-testing/src/main/java/org/sdase/commons/server/weld/testing/WeldAppExtension.java

```diff
@@ -16,7 +16,11 @@
  *   static final WeldAppExtension&lt;AppConfiguration%gt APP =
  *       new WeldAppExtension&lt;&gt;(WeldExampleApplication.class, resourceFilePath("test-config.yaml"));
  * </pre>
+ *
+ * @deprecated Deprecated in favour of the official {@link
+ *     org.jboss.weld.junit5.WeldJunit5Extension}.
  */
+@Deprecated(forRemoval = true)
```

Comment:
For the Wiremock upgrade we marked the old extension as deprecated in the main branch and removed it for v6. What do you think? This would save us some lines of code to maintain.